tests/datasource/fluent/data_asset/test_sql_asset.py

```diff
@@ -12,92 +12,86 @@
 from great_expectations.datasource.fluent.sql_datasource import _SQLAsset
 
 
-@pytest.mark.unit
-def test_add_batch_definition_fluent_sql__add_batch_definition_whole_table():
-    # arrange
-    mock_datasource = create_autospec(SQLDatasource)
+@pytest.fixture
+def datasource() -> Mock:
```

Comment:
Non-blocking, but I think the best practice here is to use mocker.

Reply:
yep! 😄 was replaced in [166c692](https://github.com/great-expectations/great_expectations/pull/9732/commits/166c692eb6202d62f0382467c8f9ff4e9c53f330)